.gitignore

```diff
@@ -1,3 +1,4 @@
 .DS_Store
 *.pyc
 env
+*.swp
```

Comment:
LOL I have this in my global gitignore, so it never came up.

Please make it `*.sw?`, though 🙏🏼